lib/matplotlib/lines.py

```diff
@@ -66,7 +66,7 @@ def _get_dash_pattern(style):
 
 
 def _scale_dashes(offset, dashes, lw):
-    if rcParams['_internal.classic_mode']:
+    if not rcParams['lines.scale_dashes']:
```

Comment:
@efiring Just making sure you are aware of this change (and it did not get lost in the noise)